src/toxcore/messenger/packet/group/mod.rs

```diff
@@ -0,0 +1,80 @@
+/*! The implementation of group chat packets.
+*/
+
+mod invite;
+
+pub use self::invite::*;
+
+use nom::be_u8;
+use crate::toxcore::binary_io::*;
+use crate::toxcore::crypto_core::*;
+
+/// Length of group chat unique bytes
+pub const GROUP_UID_BYTES: usize = 32;
+
+/// Unique id used in group chat
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct GroupUID([u8; GROUP_UID_BYTES]);
+
+impl GroupUID {
+    /// Create new object
+    pub fn new() -> Self {
```

Comment:
I'd suggest to rename it to `random`, otherwise we will have to add `Default` impl but it doesn't really fit if returns only random values.

Reply:
See https://github.com/tox-rs/tox/blob/5d2c2fcd0e32931d294a97389461f8fc710b4744/src/toxcore/tcp/secure.rs#L75